roles/branch-validation/tasks/main.yml

```diff
@@ -68,17 +61,14 @@
     nics:
       - net-name: internal
     name: "validate-{{ item }}"
+    availability_zone: "nova:{{ item }}"
     wait: yes
   with_items: groups[instance_group]
```

Comment:
Was there a strong reason for defining and using "instance_group" rather than just using `groups['compute']` ?  We just use the `compute` group in site.yml, lots would break if that's not the group where instances go.